ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -364,3 +426,8 @@ message DamlContractKeyState {
   // If unset the key is inactive.
   DamlContractId contract_id = 1;
 }
+
+// Participant not authorized to submit the request, a common rejection reason.
```

Comment:
I am not sure what pulling up just one error buys us. It seems we lose on the type safety side by gaining some ability to write reusable error handlers.

Reply:
It also isn't going to play nicely if we modularize this. I'll revert the changes.